gh-pages/_index.html

```diff
@@ -0,0 +1,472 @@
+<h2 id="plugins">
+  Plugins
+</h2>
+<p>
+  Plugins are NPM packages that extend or modify Blockly's functionality and can be included in any Blockly-based project.
+</p>
+<br>
+<h3 id="fields">
+  Fields
+</h3>
+<p>
+  Field plugins add new kinds of fields that can be incorporated into custom blocks, allowing your users to enter information in different ways. Learn more about how to <a href="https://developers.google.com/blockly/guides/create-custom-blocks/fields/customizing-fields/overview">create and customize fields</a>.
+</p>
+<div class="grid">
+  <a class="card" href="plugins/field-bitmap/test/index.html">
+    <img srcset="media/bitmap-field.png 2x" alt="">
```

Comment:
Add "a screenshot of the bitmap field plugin" here, and similar for others, as the alt text.

Reply:
Per https://www.w3.org/WAI/tutorials/images/decorative/, I think leaving it empty is generally the recommended approach in this case, since the adjacent title/description of the plugin provide information about it.